perfkitbenchmarker/pkb.py

```diff
@@ -624,18 +629,19 @@ def _GenerateBenchmarkDocumentation():
                            windows_benchmarks.BENCHMARKS):
     benchmark_config = configs.LoadMinimalConfig(
         benchmark_module.BENCHMARK_CONFIG, benchmark_module.BENCHMARK_NAME)
-    vm_groups = benchmark_config['vm_groups']
```

Comment:
It would be nice if the benchmark requirements mentioned the spark cluster and/or the VMs that constitute PKB managed clusters. Maybe we should hold off until more service types are supported so we can do something more generic